frontend/src/core/comments/components/Comment.css

```diff
@@ -5,7 +5,11 @@
 .comments-list .comment a {
     color: #7BC876;
     font-weight: 400;
-    white-space: nowrap;
+}
+
+.comments-list .comment a[href^="/contributors/"]:not(.comment-author) {
+    background-color: rgba(51, 57, 65, .4);
```

Comment:
What if we just use `#3f4752`? I don't think we need transparency and it will be easier for theming to reuse colors.